server/src/controllers/Venue/resolver.ts

```diff
@@ -6,12 +6,34 @@ import { Venue } from '../../graphql-types';
 import { prisma } from '../../prisma';
 import { CreateVenueInputs, UpdateVenueInputs } from './inputs';
 
+const VenuesData = {
```

Comment:
```suggestion
const venueIncludes = {
```
For consistency with the rest of the codebase.